llvm/lib/Transforms/IPO/SampleProfile.cpp

```diff
@@ -2498,25 +2607,41 @@ void SampleProfileMatcher::runOnModule() {
 
     SmallVector<std::pair<StringRef, uint64_t>> ProfStatsVec;
     if (FunctionSamples::ProfileIsProbeBased) {
-      ProfStatsVec.emplace_back("NumMismatchedFuncHash", NumMismatchedFuncHash);
+      ProfStatsVec.emplace_back("NumStaleProfileFunc", NumStaleProfileFunc);
```

Comment:
Does changing those strings require changing how we aggregates stats internally for dashboard etc?

Reply:
Yep, the logging is fine, we aggregate by the key without checking a specific stats, so it shouldn't break our internal build. Just for some offline tools, the last step to show the percentage things would be error, and hopefully caught by the exception. I will make some updates to those usages.

Reply:
> Yep, the logging is fine, we aggregate by the key without checking a specific stats, so it shouldn't break our internal build. Just for some offline tools, the last step to show the percentage things would be error, and hopefully caught by the exception. I will make some updates to those usages.

We want to make sure there is continuity in staleness data, and we should not lose data before the name changes.